mx.truffleruby/suite.py

```diff
@@ -290,10 +306,33 @@
 
         # ------------- Distributions -------------
 
+        "TRUFFLERUBY-ANNOTATIONS": {
+            "dependencies": [
+                "truffleruby-annotations"
+            ],
+            "description": "TruffleRuby Annotations",
+            "license": ["EPL-1.0"]
+        },
+
+        "TRUFFLERUBY-SHARED": {
```

Comment:
It would be good to add a comment this is needed to share code between the launcher and the rest, and the rest cannot depend on the launcher. Also, maybe something about this code might be loaded twice in different classloaders, so it probably shouldn't create instances to pass around, etc.